### PR TITLE
Customizatior to hero & footer, optimatization

### PR DESCRIPTION
**Pull request by Eskilmar**

### CODE REVIEW

File: src/components/pages/WelcomePage.tsx [hunk 1]
```diff
@@ -75,35 +75,34 @@ class WelcomePage extends React.Component<Props, State> {
...
+
+    this.setState({
+      customizeFields: customizeFields
+    });
+
+    const posts = await api.getWpV2Posts({per_page: 90});
+    const mainMenu = await api.getMenusV1LocationsById({ lang: this.props.lang, id: "main" });
+    const localeMenu = await api.getMenusV1LocationsById({ lang: this.props.lang, id: "locale" });
+    const popularCategory = await api.getWpV2Categories({ slug: ["suosittu"] });
+    const media = await api.getWpV2Media({per_page: 50});
```
Comment (terolamminpaa) [inline src/components/pages/WelcomePage.tsx:93]: Why make all calls separate? If you wish to get customize fields first you could still fetch everything else after that in Promise.all(). I think this version now waits between each call.

File: src/components/pages/WelcomePage.tsx [hunk 2]
```diff
@@ -156,24 +161,51 @@ class WelcomePage extends React.Component<Props, State> {
...
           </div>
           <div className= { classes.postsColumn }>
             <h1>{ <JobsIcon/> } Työpaikat</h1>
-            { this.renderAnnouncements(this.state.announcementsCategoryId) }
+            { this.state.loading &&
+              <div className={ classes.loadingIconContainer }>
+                <CircularProgress />
+              </div>
+            }
+            {!this.state.loading &&
```
Comment (terolamminpaa) [inline src/components/pages/WelcomePage.tsx:200]:
```suggestion
            { !this.state.loading &&
```

File: src/components/pages/WelcomePage.tsx [hunk 3]
```diff
@@ -293,24 +325,25 @@ class WelcomePage extends React.Component<Props, State> {
...
   }
 
   /**
-   * Returns post featured image URL
+   * Returns page featured image URL
    */
-  private getAttachmentForPost = (post: Post) => {
+  private getAttachmentForPage = (page: Page) => {
     let attachmentUrl = "";
+    console.log("URL: ",this.state.media);
```
Comment (terolamminpaa) [inline src/components/pages/WelcomePage.tsx:343]:
```suggestion
```

File: src/components/pages/WelcomePage.tsx (hunk 1 above)
Comment (Eskilmar) [reply src/components/pages/WelcomePage.tsx]: I've pushed new commit, where it's done how suggested

**Pull request merged**